### PR TITLE
Added tables migrations

### PR DESCRIPTION
**Pull request by narensrini**

### CODE REVIEW

**Comment by rossem** [comment]
Metrics?

File: db/migrations/0003_add_frames_table.up.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE frames (
```
Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql:1]: Add previous and next frame hashes. Also probably need to store the hash for this frame.

Comment (rossem) [inline db/migrations/0003_add_frames_table.up.sql:1]: do we need to have next? Id like frames to be immutable and if we have next it wont be

File: db/migrations/0003_add_frames_table.up.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  id          uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+  content     text,
+  editor      text NOT NULL,
```
Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql]: Can be null

Comment (rossem) [inline db/migrations/0003_add_frames_table.up.sql]: how

Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql]: It's just extra info, we don't know 100% that we're gunna need it.

File: db/migrations/0003_add_frames_table.up.sql (hunk 1 above)
Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql:1]: Sounds good. Why does it need to be hashes, can't we just store the uuid of the previous frame?

Comment (rossem) [inline db/migrations/0003_add_frames_table.up.sql:1]: Yeah I just thought of that as well. But then we need to generate the uuids on the client side, which seems kinda sketch for a primary key lol. I think its fine with hashes

Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql:1]: True, just realized that as well. How are we gunna deal with collisions?

Comment (rossem) [inline db/migrations/0003_add_frames_table.up.sql:1]: ¯\_(ツ)_/¯ its extremely unlikely we gonna have the same hash for the same file

Comment (Wojtechnology) [inline db/migrations/0003_add_frames_table.up.sql:1]: ... i guess. Thats why i liked the increment idea. We can experiment

**Comment by rossem** [comment]
i made some changes

**Comment by Wojtechnology** [comment]
👍👍👍👍👍👊👊👊💪💪💪

**Comment by rossem** [comment]
:trumpet: :trumpet: :trumpet: :trumpet:

**Pull request merged**